modules/googleplay/src/main/scala/cards/nine/googleplay/processes/CardsProcesses.scala

```diff
@@ -180,13 +181,13 @@ class CardsProcesses[F[_]](
       // Does package exists in Google Play?
       webScrapper.existsApp(pack) flatMap {
         case true ⇒
-          cacheService.setToPending(pack).map(_ ⇒ PendingResolution(pack))
+          toFree(cacheService.setToPending(pack).map(_ ⇒ PendingResolution(pack)))
         case false ⇒
-          cacheService.addError(pack).map(_ ⇒ UnknownPackage(pack))
+          toFree(cacheService.addError(pack).map(_ ⇒ UnknownPackage(pack)))
```

Comment:
If I understand this correctly `toFree` is just an interim conversion until all algebras are migrated to Freestyle right?